src/Common/memcmpSmall.h

```diff
@@ -625,6 +661,9 @@ inline bool memequalSmallAllowOverflow15(const Char * a, size_t a_size, const Ch
     if (a_size != b_size)
         return false;
 
+    __msan_unpoison(&a[a_size - a_size % 16], a_size % 16);
```

Comment:
can you help me understand why this part is different than others which use `__msan_unpoison_overflow_15`?

Reply:
Because I missed it. First I did it manually (and wrong) and then decided it was much better to have a single macro to do it, but missed a replacement. I'll fix it.